index.js

```diff
@@ -1,60 +1,50 @@
-let books = [];
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
 
-// form declaration
+const books = JSON.parse(localStorage.getItem('books')) || [];
 
-const title = document.querySelector('#title');
```

Comment:
- [ ] Great job so far 👏🏻 However, for this milestone, you should use JavaScript `classes` instead of `objects` and `arrays`, please note that you can use objects and arrays but it has to be **inside** the classes.
- Please, check this example of how it can be implemented.
```javascript
class Books {
  constructor() {
    this.books = [];
  }

 // It must have class `methods` to add and remove books

  addBook(title, author) {
    ...
    return this.books;
  }

  removeBook(bookID) {
    this.books = this.books.filter(({ id }) => bookID !== id);
   ...
  }
}
```

- [ ] Kindly make sure  It keeps all functionality of the application. The refactor don't have to introduce bugs to the app and all functions from previous project must still be working fine. 💯 👍🏻